src/main/java/net/atomique/ksar/UI/GraphView.form

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.1" encoding="UTF-8" ?>
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I think the migration efforts will be high. Or at least "more than zero". And we cannot exclude *.form files in the repository, because IDE always generated corresponding Java files from them. I'd say that we do not need Java files because of this fact :)